src/mono/wasm/Wasm.Build.Tests/ProjectProviderBase.cs

```diff
@@ -374,6 +374,9 @@ private string[] GetFilesMatchingNameConsideringFingerprinting(string filePath,
         if (IsFingerprintingEnabled)
         {
             string bootJsonPath = Path.Combine(paths.BinFrameworkDir, "dotnet.boot.js");
+            if (!File.Exists(bootJsonPath))
+                bootJsonPath = Path.Combine(paths.BinFrameworkDir, "dotnet.js"); // inline boot config
```

Comment:
Won't this be confusing that we are checking is-config-inlined by the presence of config files, not by `WasmInlineBootConfig` property? Can it happen that "dotnet.boot.js" won't be produced because of runtime logic error, not because inlining was enabled?

Reply:
Ideally, we should pass `MSBuildOptions` (or something similar) here as to other places check the boot config file name.
I can make this refactoring in a follow-up.